core/activity/src/http_proxy.rs

```diff
@@ -44,9 +44,12 @@ async fn proxy_http_request(
     let activity_id = path_activity_url.activity_id;
     let path = path_activity_url.url;
 
-    let result = authorize_activity_executor(&db, id.identity, &activity_id, Role::Requestor).await;
+    let result =
+        authorize_activity_initiator(&db, id.identity, &activity_id, Role::Requestor).await;
```

Comment:
Does this change alter semantics anyhow?

Reply:
This is a bug, because wrong identity is checked: https://github.com/golemfactory/ya-runtime-ai/issues/105

Reply:
Then IMO we should use merge without squash so that the commit message for the fix stays separate.

Reply:
I always use merge for my PRs